src/posix/http_client_curl.cc

```diff
@@ -41,12 +44,15 @@ std::string RunCurl(const std::string& cmd) {
   FILE* pipe = ::popen(cmd.c_str(), "r");
   assert(pipe);
   char s[8 * 1024];
-  ::fgets(s, sizeof(s) / sizeof(s[0]), pipe);
+  std::string result;
+  while (nullptr != ::fgets(s, sizeof(s) / sizeof(s[0]), pipe)) {
+    result += s;
```

Comment:
IIRC, this might result in newlines inconsistency. Of two sorts:

1. `"\r\n"` vs `"\n"`, and
2. The last line will not end with the newline character, if it was not in the input data.

Would you like to address it, or not now?